schema_based_on_diagram.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE patients (
+  id                SERIAL PRIMARY KEY,
+  name              VARCHAR,
+  date_of_birth     DATE
+);
+
+CREATE TABLE medical_histories (
+  id                SERIAL PRIMARY KEY,
+  admitted_at       TIMESTAMP,
+  patient_id        INT REFERENCES patients(id),
+  status            VARCHAR
+)
+;
+CREATE TABLE invoices (
+  id                SERIAL PRIMARY KEY,
+  total_amount      DECIMAL,
+  generated_at      TIMESTAMP,
+  payed_at          TIMESTAMP,
+  medical_history_id INT REFERENCES medical_histories(id)
+);
+
+CREATE TABLE treatments (
+  id                INT PRIMARY KEY REFERENCES medical_histories(id),
+  type              VARCHAR,
+  name              VARCHAR,
+);
+
+CREATE TABLE invoice_items (
+  id                SERIAL PRIMARY KEY,
+  unit_price        DECIMAL,
+  quantity          INT,
+  total_price       DECIMAL,
+  invoice_id        INT REFERENCES invoices(id),
+  treatment_id      INT REFERENCES treatments(id)
```

Comment:
- [ ] Please create a table between medical_histories and treatments it can be like this:
```sql
CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```